db/seeds.rb

```diff
@@ -8,6 +8,16 @@ def generate_base_url(env_name)
 end
 
 if Rails.env.development?
+  [
```

Comment:
Should have some way to seed/import set of common permissions for existing endpoints.